test/acceptance/features/bindAppToService.feature

```diff
@@ -6,9 +6,8 @@ Feature: Bind an application to a service
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
-        * PostgreSQL DB operator is installed
+        # * PostgreSQL DB operator is installed
```

Comment:
please remove if not needed.

Reply:
Sorry commenting it is a mistake. Reverting it back.